desktop_version/src/Graphics.cpp

```diff
@@ -2268,8 +2266,18 @@ void Graphics::drawtowerbackground()
         ScrollSurface(towerbuffer, 0, -map.bscroll);
         for (int i = 0; i < 40; i++)
         {
-            temp = map.tower.backat(i, 0, map.bypos);
-            drawtowertile3(i * 8, -(map.bypos % 8), temp, map.colstate);
+            if (map.scrolldir == 0)
```

Comment:
Should we put this branch outside of the loop?

Reply:
If it's faster, then sure.

Reply:
Done, rebased and force-pushed.